jfix-armeria-facade/src/main/kotlin/ru/fix/armeria/facade/webclient/HttpClientBuilderApi.kt

```diff
@@ -18,7 +18,7 @@ import java.time.Duration
 interface BaseHttpClientBuilder<out HttpClientBuilderT : BaseHttpClientBuilder<HttpClientBuilderT>> {
 
     /*
-     * Mandatory builder methods. Client name and endpoint/endpoint_group must be formed.
+     * Methods could be omitted. If so, default WebClient without endpoint_group/endpoint will be built
      */
 
     fun setClientName(clientName: String): HttpClientBuilderT
```

Comment:
`clientName` is mandatory, see `throw IllegalStateException("clientName must be set")`

also there is not relevant comment on lines 60 - 62
```
   /*
     * END Mandatory builder methods.
     */
```